ArrayEx/House.java

```diff
@@ -0,0 +1,26 @@
+package ArrayEx;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by LewisMonplaisir on 17/07/2017.
+ */
+public class House {
+    public void HouseTypes() {
```

Comment:
method names shoud not start with uppercase in Java. Take note of this but it is nothing so serious now.